core/citrus-base/src/test/java/com/consol/citrus/functions/core/ReadFileResourceFunctionTest.java

```diff
@@ -31,34 +33,38 @@
 public class ReadFileResourceFunctionTest extends UnitTestSupport {
 
     /** Class under test */
-    private ReadFileResourceFunction function = new ReadFileResourceFunction();
+    private final ReadFileResourceFunction function = new ReadFileResourceFunction();
 
     @Test
-    public void testExecute() throws Exception {
+    public void testExecute()  {
         context.setVariable("filename", "file.txt");
         context.setVariable("user", "Christoph");
 
         String path = "classpath:com/consol/citrus/functions/${filename}";
-        String result = function.execute(Arrays.asList(path), context);
+        String result = function.execute(Collections.singletonList(path), context);
```

Comment:
I thought `List.of` is better use.. but I cannot find sources :(

Reply:
It boils down to: "it's a matter of choice."
https://dzone.com/articles/singleton-list-showdown-collectionssingletonlist-v

I am used to use singletonList because of java 1.8 - thats all. Since you brought up the matter i lookup into the code. I changed that piece of code because of a warning on the usage of Arrays.asList with only a single parameter. I can use List.of in both places of the code without a warning. This makes it consistent.